### PR TITLE
Add support for Amazon ECS services

### PR DESCRIPTION
**Pull request by adamlogic**

### CODE REVIEW

File: judoscale-ruby/lib/judoscale/config.rb [hunk 1]
```diff
@@ -89,9 +89,11 @@ def reset
         @api_base_url ||= "https://adapter.judoscale.com/api/#{ENV["RENDER_SERVICE_ID"]}"
       elsif ENV["DYNO"]
         @current_runtime_container = RuntimeContainer.new ENV["DYNO"]
+      elsif (metadata_uri = ENV["ECS_CONTAINER_METADATA_URI"])
+        @current_runtime_container = RuntimeContainer.new(metadata_uri.split("/").last)
```
Comment (jon-sully) [inline judoscale-ruby/lib/judoscale/config.rb:93]: Ah, I totally missed this PR email and read my emails in backwards order. This makes more sense now!

File: judoscale-ruby/test/config_test.rb [hunk 2]
```diff
@@ -60,6 +60,20 @@ module Judoscale
       end
     end
 
+    it "initializes the config from default Amazon ECS ENV vars" do
+      # Users must set JUDOSCALE_URL manually when using Amazon ECS.
+      env = {
+        "ECS_CONTAINER_METADATA_URI" => "http://169.254.170.2/v3/a8880ee042bc4db3ba878dce65b769b6-2750272591",
```
Comment (jon-sully) [inline judoscale-ruby/test/config_test.rb:66]: Seems obvious but just want to make sure our bases are covered — the last value there _is_ globally unique to separate tasks spun up under the same service umbrella?

Comment (adamlogic) [reply judoscale-ruby/test/config_test.rb:66]: As far as I can tell, yes, but I'm not 100% sure. We don't actually use it for anything publicly available, though, so it's kind of arbitrary.

Comment (jon-sully) [reply judoscale-ruby/test/config_test.rb:66]: Works for me!

**Review by jon-sully — APPROVED** [review]

**Pull request merged**